test/polarLine2.html

```diff
@@ -26,12 +26,17 @@
     </head>
     <body>
         <style>
-            html, body, #main {
+            html, body, .test-chart {
                 width: 100%;
                 height: 100%;
             }
         </style>
-        <div id="main"></div>
+        <h1>main</h1>
```

Comment:
Please do not change in this file because this is for line series in polar system. Consider moving to `bar-polar-stack.html`.